common/ethnicities/wc_ethnicities_elven.txt

```diff
@@ -79,30 +79,6 @@ high_elven = {
     # }
```

Comment:
I would say that the chin problem still exists... or more like it's the jaw angle problem.
Here I compared an elf and human.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104062350-be919400-5213-11eb-99f0-21985408c5e9.png)
![image](https://user-images.githubusercontent.com/46384992/104062462-f8629a80-5213-11eb-9582-68da431f0a1c.png)

</details>